hyperspy/models/edsmodel.py

```diff
@@ -318,6 +318,62 @@ def disable_xray_lines(self):
         for component in self._active_xray_lines:
             component.active = False
 
+    def fit(self, fitter='mpfit', method='ls', grad=True,
+            bounded=True, ext_bounding=False, update_plot=False,
+            **kwargs):
+        """Fits the EDS model to the experimental data.
+
+        The fitting of an EDS model is, by default, performed by bounded
+        optimisation using mpfit. This constrains the components to be positive
+        as is physical. Alternative fitters may still be specified.
+
+        Parameters
+        ----------
+        fitter : {None, "leastsq", "odr", "mpfit", "fmin"}
+            The optimizer to perform the fitting. By default mpfit is used for
+            EDS models, as this supports bounding to force the components in the
+            model to be positive. This is inline with the physical nature of EDS
+            spectra. All optimizers available for Model1D remain availble and
+            can be specified by the user. fmin is the only one that supports
+            maximum likelihood estimation, but it is less robust than leastsq
```

Comment:
Surely the thing to do here is exactly what we would do in any other academic context and insist on referencing appropriate literature that actually demonstrates this or at least gives some evidence...

Who says Levenberg-Marquardt turns out to be pretty good at it? And what does that even mean?